src/client/app/components/about.ts

```diff
@@ -50,10 +64,31 @@ export class About implements AfterViewInit {
     currentChosenLanguage = '';
     originalLanguage = '';
 
+    configs = [];
+    selectedConfig = '';
+
+    constructor(
+        private _effects: UIEffects,
+        private _changeDetector: ChangeDetectorRef
+    ) {}
+
     ngAfterViewInit() {
         this.availableLanguages = getAvailableLanguages();
         this.currentChosenLanguage = getDisplayLanguage();
         this.originalLanguage = this.currentChosenLanguage;
+        this.configs.push(
+            { name: 'production', value: 'Production' },
```

Comment:
Nit: I wonder if you use the word "name" and "value" differently here than is customary.  Value is generally the programmatic name, whereas name is the display string.  I think you'd doing the opposite here.

E.g., instead of

    <option *ngFor="let c of configs" [value]="c.name">{{c.value}}</option>

It should really be:

    <option *ngFor="let c of configs" [value]="c.value">{{c.name}}</option>